versions/1.0.0.md

```diff
@@ -113,7 +113,7 @@ Field Name | Type | Description
 ---|:---:|---
 <a name="actionTarget"></a>target | `string` | **REQUIRED** A JSONPath query expression referencing the target objects in the target document.
 <a name="actionDescription"></a>description | `string` | A description of the action. [CommonMark syntax](https://spec.commonmark.org/) MAY be used for rich text representation.
-<a name="actionUpdate"></a>update | Any | An object with the properties and values to be merged with the object(s) referenced by the `target`. This property has no impact if `remove` property is `true`.
+<a name="actionUpdate"></a>update | Any | If the `target` is an object, an object with the properties and values to merge with the `target`. If the `target` is an array, an entry to append to the array. To replace an array in its entirety, apply a remove action followed up an update action. This property has no impact if `remove` property is `true`.
```

Comment:
I think this perhaps requires more context below this table where we can create additional prose. I'm wondering if it should be reworked to give us more space for explanation.

Apologies for being pedantic below.

- `target` needs to be better defined in this context.  Is it the JSONPath query itself? Is it the document the query is using as its source?  Is it the result of the query on the document? What happens if a result contains multiple entries of mixed types?
- JSON has "members" or "name/value pairs", not "properties".
- "To replace an array in its entirety, apply a remove action followed up an update action. This property has no impact if `remove` property is true." This is a little confusing.  I think it means "followed _by_ an update action". The last sentence seemingly contradicts the first, and I think it can be rephrased with more clarity.

I would propose keeping it short here and adding more detail below where we can expand beyond the table format.

Reply:
Easiest question first, the `target` is the thing two lines up, the target of this action.

I definitely didn't nail the field/property wording, I'll give that another go.